### ISSUE TITLE
Multiple container log in one window

### ISSUE THREAD

**Issue by webysther** [amir20/dozzle#2170]
The log view in docker compose for the stack is fantastic but in so many cases not available, if a tool like this offer the view option for stack, I don't know if is possible to detect stack from docker compose and show the logs for all running containers, but the unified view by choosing the containers will be a great add.

![image](https://user-images.githubusercontent.com/750007/236017412-b954a17e-9078-44c3-a2cd-db4c8045440a.png)

As a expansion of the initial ideia, maybe enable creating 'playlists' with selected containers to save and make simple to have a stack-like view no matter when started the container.

**Comment by amir20** [comment]
This was also discussed in #2150

**Comment by amir20** [comment]
So I like this option more compared to #2150 because in the previous request, I had to think about all the right UI to make it happen. I am not a big fan of creating more buttons, more menus or undiscoverable mouse shortcuts.  

This would be simpler to show because then on the left it would have a list of all the "containers" and "stacks". Maybe even "services". 

>  I don't know if is possible to detect stack from docker compose and show the logs for all running containers, but the unified view by choosing the containers will be a great add.

I did some research and it seems like the closest thing to getting the stack is through labels for each container.  

But there is a bigger problem that needs to be resolved first. Dozzle _doesn't_ support services right now. To do this correctly, Dozzle would need to first support services across multiple nodes. Services use a different API. But last I checked, the API for service logs was very buggy.  Related issues #1196 #961 and of course most importantly #297. Last I checked https://github.com/moby/moby/issues/33183 there were issues fetching all logs across multiple instances which means Dozzle would also have those bugs.  I assume your use case is a single node swarm host. So one idea would be to explicitly only support 

So in summary, to support stacks, services need to be supported first. To support services, swarm and logs across multiple nodes need to be supported which seems buggy. 

For those reasons, I haven't implemented any of the swarm APIs.
